### PR TITLE
Show logs when terraform fails in `lokoctl cluster apply/destroy` 

### LINKED ISSUE #225: Improve terraform related UX
It is hard to know what's wrong when something fails in a terraform step.

We should:

- [ ] Tell the user where the terraform log is when running without `--verbose`
- [ ] In case of error, show the last lines of the terraform output

Reported-by: @jpetazzo

### PR DESCRIPTION
**Pull request by surajssd**
When user is using lokoctl cluster apply/destroy in normal mode (not verbose) then show users what went wrong by showing last 20 lines of the logs.

And also let user know where the log file resides.

Fixes #225

### CODE REVIEW

File: pkg/terraform/executor.go [hunk 1]
```diff
@@ -211,22 +213,50 @@ func (ex *Executor) execute(verbose bool, args ...string) error {
 		wg.Add(1)
 
 		go tailFile(p, done, &wg)
+	} else {
+		fmt.Printf("\nYou can find the logs in %q\n", p)
```
Comment (invidian) [inline pkg/terraform/executor.go:217]: I'm not really sure if this is needed. Also, it uses `else` :smile:

File: pkg/terraform/executor.go [hunk 2]
```diff
@@ -211,22 +213,50 @@ func (ex *Executor) execute(verbose bool, args ...string) error {
...
+			showError(p, noOfLinesOnError)
+		}
 		return fmt.Errorf("executing Terraform failed, check %s for details", p)
 	}
 
 	return nil
 }
 
+func showError(path string, noOfLines int) {
+	data, err := ioutil.ReadFile(path)
```
Comment (invidian) [inline pkg/terraform/executor.go:241]: Needs `#nosec G304`.

File: pkg/terraform/executor.go [hunk 3]
```diff
@@ -211,22 +213,50 @@ func (ex *Executor) execute(verbose bool, args ...string) error {
...
+	if err != nil {
+		fmt.Printf("error reading file: %v", err)
+		return
+	}
+	text := string(data)
+
+	lines := strings.Split(text, "\n")
+	offset := len(lines) - noOfLines - 1
```
Comment (invidian) [inline pkg/terraform/executor.go:249]:
```suggestion
	offset := len(lines) - noOfLines - 1 //nolint:gomnd
```

Comment (invidian) [reply pkg/terraform/executor.go:249]: Actually, why 1? :smile:

File: pkg/terraform/executor.go (hunk 1 above)
Comment (surajssd) [reply pkg/terraform/executor.go:217]: But then that is one of the requirement from the issue. What is wrong with using `else`?

File: pkg/terraform/executor.go (hunk 3 above)
Comment (surajssd) [reply pkg/terraform/executor.go:249]: Oh that is to adjust the numbers. So we count from 1 and computer counts from 0 so this negation makes that adjustment.

Comment (surajssd) [reply pkg/terraform/executor.go:254]: Added comment.

File: pkg/terraform/executor.go (hunk 1 above)
Comment (invidian) [reply pkg/terraform/executor.go:217]: About using `else`: https://medium.com/@jamousgeorges/dont-use-else-66ee163deac3

**Pull request merged**